src/main/java/uk/gov/hmcts/cmc/claimstore/events/claimantresponse/ClaimantResponseStaffNotificationHandler.java

```diff
@@ -36,11 +47,17 @@ public void onClaimantResponse(ClaimantResponseEvent event) {
         if (isResponseFullDefenceStatesPaid(response)) {
             this.statesPaidStaffNotificationService.notifyStaffClaimantResponseStatesPaidSubmittedFor(claim);
         }
-        if (isPartAdmission(response)
-            && claim.getClaimantResponse().orElseThrow(IllegalArgumentException::new).getType()
-            == ClaimantResponseType.REJECTION) {
+        ClaimantResponse claimantResponse = claim.getClaimantResponse().orElseThrow(IllegalArgumentException::new);
+        if (isPartAdmission(response) && claimantResponse.getType() == ClaimantResponseType.REJECTION) {
             claimantRejectionStaffNotificationService.notifyStaffClaimantRejectPartAdmission(claim);
         }
+        if (isReferredToJudge(claimantResponse)) {
```

Comment:
The nested if can be avoided with

```suggestion
        if (!isReferredToJudge(claimantResponse)) {
            return;
        }
        
         if (isCompanyOrOrganisation(response.getDefendant())) {
         ....
```

Reply:
I'd prefer not to explicitly `return` here, in case we end up adding new claimant-response conditions under which a staff notification should be sent.